lib/irb.rb

```diff
@@ -887,49 +830,7 @@
 #
 # === Commands
 #
-# The following commands are available on IRB.
-#
-# * cwws
-#   * Show the current workspace.
-# * cb, cws, chws
-#   * Change the current workspace to an object.
-# * bindings, workspaces
-#   * Show workspaces.
-# * pushb, pushws
-#   * Push an object to the workspace stack.
-# * popb, popws
-#   * Pop a workspace from the workspace stack.
-# * load
-#   * Load a Ruby file.
-# * require
-#   * Require a Ruby file.
-# * source
-#   * Loads a given file in the current session.
-# * irb
-#   * Start a child IRB.
-# * jobs
-#   * List of current sessions.
-# * fg
-#   * Switches to the session of the given number.
-# * kill
-#   * Kills the session with the given number.
-# * help
-#   * Enter the mode to look up RI documents.
-# * irb_info
-#   * Show information about IRB.
-# * ls
-#   * Show methods, constants, and variables.
-#     -g [query] or -G [query] allows you to filter out the output.
-# * measure
-#   * measure enables the mode to measure processing time. measure :off disables it.
-# * $, show_source
-#   * Show the source code of a given method or constant.
-# * @, whereami
-#   * Show the source code around binding.irb again.
-# * debug
-#   * Start the debugger of debug.gem.
-# * break, delete, next, step, continue, finish, backtrace, info, catch
-#   * Start the debugger of debug.gem and run the command on it.
+# Please use the `show_cmds` command to see the list of available commands.
```

Comment:
We need to find a way to sync commands into the doc here. But for now, users can find them through readme or through the `show_cmds` command.